imports/blocks/give/index.js

```diff
@@ -42,7 +38,6 @@ const withCheckout = graphql(CHECKOUT_QUERY, {
   options: { variables: { state: 28, country: 45 } },
```

Comment:
let's setup the defined values to be the maps needed for inputs as a `props` function. Should be more performant and a good practice to start as it's easier to share